### PR TITLE
fix: V3 notification visual changes

### PR DESCRIPTION
**Pull request by burakukula**
# Purpose of PR

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

**Review by suevalov — APPROVED** [review]

File: packages/forma-36-react-components/src/components/Notification/Notification.stories.tsx [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
-import { button, text, number, boolean, select } from '@storybook/addon-knobs';
 
 import Button from '../Button';
-import Notification from './index';
-import notes from './Notification.md';
+import Notification, { NotificationsAPI } from './index';
```
Comment (gui-santos) [inline packages/forma-36-react-components/src/components/Notification/Notification.stories.tsx:4]: I know it's not a change that you made but... Could you rename NotificationAPI to NotificationProps in this PR, pls?
Just so it follows the naming convention we are using in the other components 

🙏

**Review by gui-santos — APPROVED** [review]
🚢

File: packages/forma-36-react-components/src/components/Notification/Notification.stories.tsx (hunk 1 above)
Comment (burakukula) [reply packages/forma-36-react-components/src/components/Notification/Notification.stories.tsx:4]: @gui-santos as we export this name (most likely is not used anywhere), would it be in theory, a breaking change if we change the namespace?

Comment (gui-santos) [reply packages/forma-36-react-components/src/components/Notification/Notification.stories.tsx:4]: I think it's only used here, but I see your point 👍 
let's do it later then

**Review by fabe — APPROVED** [review]

**Pull request merged**